src/entities/viewer/index.ts

```diff
@@ -0,0 +1 @@
+export * from './model/api';
```

Comment:
use namespace for model
export * as viewerModel from .... etc

Reply:
fixed

Reply:
fixed

Reply:
model/api should not be exported, it's for internal use